build.gradle

```diff
@@ -189,10 +189,16 @@ dependencies {
     implementation "io.swagger.parser.v3:swagger-parser-core:${swaggerVersion}"
     implementation "io.swagger.parser.v3:swagger-parser:${swaggerVersion}"
     implementation "io.swagger.parser.v3:swagger-parser-v3:${swaggerVersion}"
-    implementation "com.fasterxml.jackson.core:jackson-core:${jacksonVersion}"
```

Comment:
Not sure why core was added twice here and [here](https://github.com/opensearch-project/flow-framework/pull/912/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R212)

Reply:
> Not sure why core was added twice here and [here](https://github.com/opensearch-project/flow-framework/pull/912/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R212)

I commented on the `main` PR here, there are different transient dependencies that I had to address with https://github.com/opensearch-project/flow-framework/pull/906/commits/d3c8141557bf5799559fa31bb37b652198909cb2

slf4j is included with the rest client in 3.x but not in 2.x.

Similarly with Jackon core in the x-content class.

I haven't dug into the details but adding the dependency fixed the runtime errors like this:
```
RegisterAgentTests > testApiSpecRegisterAgentInputParamComparison FAILED
    java.lang.NoSuchMethodError: 'void com.fasterxml.jackson.core.base.ParserMinimalBase.<init>(com.fasterxml.jackson.core.StreamReadConstraints)'
```

Feel free to propose a better solution if you know it!

Reply:
I see. Build passed with this change. So we should be good for 2.x.